app/main/forms/frameworks.py

```diff
@@ -55,6 +55,7 @@ class ReuseDeclarationForm(FlaskForm):
     reuse = DMBooleanField(
         "Do you want to reuse the answers from your earlier declaration?",
         false_values=("False", "false", ""),
+        widget=DMSelectionButtonBase(type="boolean"),
```

Comment:
If you want to retain the checkbox you can use 
```suggestion
        widget=DMCheckboxInput(),
```

Reply:
I think we tried that and it didn't work. Will give it another whirl...

Reply:
Actually the DMBooleanField uses DMCheckboxInput by default, there may be an issue with that widget